#!/usr/env/bin python
PACKAGE = 'first_project'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator() 


gen.add("safe_param",    int_t,    0, "Min safe distance",5, 5 , 100)
gen.add("unsafe_param", int_t, 1, "Min unsafe distance",    1,1, 5)


exit(gen.generate(PACKAGE, "collector", "parameters"))
